src/task2.1/services/userGroupService.js

```diff
@@ -0,0 +1,27 @@
+import { UserGroup } from '../models';
+import dataBase from '../data-access/db';
+
+class UserGroupService {
+    constructor(db) {
+        this.db = db;
+    }
+
+    async addUsersToGroup(groupId, userIds) {
+        const t = await this.db.transaction();
+
+        const rows = userIds.map(userId => ({
+            user_id: userId,
+            group_id: groupId
+        }));
+
+        await UserGroup.bulkCreate(rows, {
+            transaction: t
+
+        })
+            .then(() => {
+                t.commit();
+            });
+    }
+}
+
+export default new UserGroupService(dataBase);
```

Comment:
Pay attention to the sequelize documentation. https://sequelize.org/master/manual/assocs.html#creating--updating-and-deleting. To make the code even prettier you can use mixins https://sequelize.org/master/manual/assocs.html#special-methods-mixins-added-to-instances